packages/xdl/src/detach/IosNSBundle.js

```diff
@@ -311,6 +311,27 @@ async function _configureInfoPlistAsync(context) {
       delete infoPlist['FacebookDisplayName'];
     }
 
+    // if the developer has set a non-empty value or if Facebook SDK will want to autoinitialize itself
+    // (given the default is to autoinitialize and all the prerequisites are there)
+    if (config.facebookAutoInitEnabled !== undefined || config.facebookAppID) {
+      infoPlist.FacebookAutoInitEnabled = config.facebookAutoInitEnabled || false;
+    } else {
+      delete infoPlist['FacebookAutoInitEnabled'];
+    }
+
+    if (config.facebookAutoLogAppEventsEnabled !== undefined) {
```

Comment:
Just to be clear, if auto-initialization is disabled (set to false), then auto-logging app events does not do anything and the Facebook SDK will not collect data?

Reply:
With all the Facebook features enabled:

![Zrzut ekranu 2019-10-15 o 14 22 51](https://user-images.githubusercontent.com/1151041/66831201-678d0880-ef57-11e9-94d8-6c0835aa7a30.png)

With all the Facebook features enabled BUT auto init:

![Zrzut ekranu 2019-10-15 o 14 22 09](https://user-images.githubusercontent.com/1151041/66831231-7378ca80-ef57-11e9-9aa9-a1f875027d74.png)